File: java/KthElement.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class KthElement {
    public static void main(String[] args) {
        findKthLargestAndSmallest(new int[]{1,9,0,10,15,2}, 2).forEach(e -> System.out.println(e));
    }
    public static List<String> findKthLargestAndSmallest(int [] nums, int k){
        List<String> result = new ArrayList<>();
        Arrays.sort(nums);
        if(k>=0 && k<nums.length){
            result.add("Min : " +nums[k-1]);
            result.add("Max : " +nums[nums.length-k]);
        }
        return result;
    }
}

File: js/kThElement.js
function findKthLargestAndSmallest(nums,kTh){
    nums.sort((a,b) => a-b);
    var res = [];

    if(nums.length > 0 && nums.length > kTh){
        res.push(` Min at ${kTh} pos : ` +nums[kTh-1]);
        res.push(` Max at ${kTh} pos : ` +nums[nums.length - kTh]);
    }
    return res;
}

const res = findKthLargestAndSmallest([1,99,50,0,23,199],3); //0,1,23,50,99,199
res.forEach(e => console.log(e));
